app/src/main/java/com/example/packpals/models/PhotoAlbum.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.packpals.models
+
+data class PhotoAlbum (
+    var albumId: String,
+    val title: String,
+    val coverPhotoUrl: Int,
```

Comment:
why is this an int

Reply:
cuz its just for hard coded data rn. Its not a URL yet

Reply:
ok should probably be a string though no?